### PR TITLE
Add Hacktober window for October 

### PR DESCRIPTION
**Pull request by nobeeakon**
hacktober modal 

imagen

changed ids for donate modal

### CODE REVIEW

File: themes/grass/layouts/index.html [hunk 1]
```diff
@@ -14,6 +14,41 @@ <h1 class="mb-3"><img alt="GRASS GIS" src="{{.Site.BaseURL}}/images/logos/grassl
       </div>
     </div>
   </header>
+
+  <div class="modal show fade" id="hacktoberfest-modal" tabindex="-1" aria-hidden="true">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-body">
+          <p>
+      <a href="https://hacktoberfest.com/"><b>Hacktober fest</b></a> is here!!, a full month to contribute to open source software. </p>
```
Comment (wenzeslaus) [inline themes/grass/layouts/index.html:23]:
```suggestion
      <a href="https://hacktoberfest.com/"><b>Hacktober fest</b></a> is here!!! A full month to contribute to open source software.</p>
```

File: themes/grass/layouts/index.html [hunk 2]
```diff
@@ -14,6 +14,41 @@ <h1 class="mb-3"><img alt="GRASS GIS" src="{{.Site.BaseURL}}/images/logos/grassl
...
+      <p>
+      Ever wanted to share your talent with 
+     GRASS GIS?, this is an excellent opportunity, coding and non coding contributions are welcomed.
```
Comment (wenzeslaus) [inline themes/grass/layouts/index.html:26]:
```suggestion
      Ever wanted to share your talent with 
     GRASS GIS? This is an excellent opportunity. Both coding and non coding contributions are welcome.
```

File: themes/grass/layouts/index.html [hunk 3]
```diff
@@ -14,6 +14,41 @@ <h1 class="mb-3"><img alt="GRASS GIS" src="{{.Site.BaseURL}}/images/logos/grassl
...
+  <div class="modal show fade" id="hacktoberfest-modal" tabindex="-1" aria-hidden="true">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-body">
+          <p>
+      <a href="https://hacktoberfest.com/"><b>Hacktober fest</b></a> is here!!, a full month to contribute to open source software. </p>
+      <p>
+      Ever wanted to share your talent with 
+     GRASS GIS?, this is an excellent opportunity, coding and non coding contributions are welcomed. 
+     Please have a look at <a href="https://github.com/OSGeo/grass/issues?q=is%3Aopen+is%3Aissue+label%3AHacktoberfest"><b>GRASS GIS repo</b></a>,
```
Comment (wenzeslaus) [inline themes/grass/layouts/index.html:27]:
```suggestion
     Please have a look at <a href="https://github.com/OSGeo/grass/issues?q=is%3Aopen+is%3Aissue+label%3AHacktoberfest"><b>GRASS GIS repo</b></a>
```

**Review by wenzeslaus — COMMENTED** [review]
Can you please include comment on the donateToGrassModal change?

Otherwise, with the suggestions accepted or otherwise addressed, I'm for merging this.

**Comment by nobeeakon** [comment]
@wenzeslaus , accepted suggestions, and included a comment in the PR description on the change for the modal ids.

**Review by neteler — APPROVED** [review]
Thanks!

**Comment by wenzeslaus** [comment]
Thanks for the idea and implementation!

**Comment by wenzeslaus** [comment]
Hacktoberfest is over. This needs a revert now (but not the donateDialog part).

**Pull request merged**